### PR TITLE
[SignInQuickstart] Update sample

### PR DESCRIPTION
**Pull request by gonzalonm**
- Updated sample to the last google android version
- Improved readme file to configure the account for sign-in with Google.

Related issue: https://github.com/xamarin/monodroid-samples/issues/187

### CODE REVIEW

File: google-services/SigninQuickstart/SigninQuickstart/SigninQuickstart.csproj [hunk 1]
```diff
@@ -139,19 +140,9 @@
     <AndroidResource Include="Resources\values-land\dimens.xml" />
     <AndroidResource Include="Resources\values-w820dp\dimens.xml" />
   </ItemGroup>
-  <Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">
```
Comment (pjcollins) [inline google-services/SigninQuickstart/SigninQuickstart/SigninQuickstart.csproj]: I'd like to retain this target to ensure these files exist before we build.

Comment (gonzalonm) [reply google-services/SigninQuickstart/SigninQuickstart/SigninQuickstart.csproj]: Done

**Comment by pjcollins** [comment]
@gonzalonm this looks great, however I think while resolving conflicts you may have dropped package references to the following:

* Xamarin.GooglePlayServices.Auth.Base.60.1142.0
* Xamarin.GooglePlayServices.Auth.60.1142.0
* Xamarin.GooglePlayServices.Auth.Api.Phone.60.1142.0

**Comment by pjcollins** [comment]
Looks great, thanks!

**Pull request merged**